Sources/HCICreateConnection.swift

```diff
@@ -0,0 +1,190 @@
+                                     packetType: HCICreateConnection.PacketType,
+                                     pageScanRepetitionMode: HCICreateConnection.PageScanRepetitionMode,
+                                     reserved: HCICreateConnection.Reserved,
+                                     clockOffset: HCICreateConnection.ClockOffset,
+                                     allowRoleSwitch: HCICreateConnection.AllowRoleSwitch,
+                                     timeout: HCICommandTimeout = .default) throws {
+        
+        let createConnection = HCICreateConnection(address: address,
+                                                   packetType: packetType,
+                                                   pageScanRepetitionMode: pageScanRepetitionMode,
+                                                   reserved: reserved,
+                                                   clockOffset: clockOffset,
+                                                   allowRoleSwitch: allowRoleSwitch)
+        
+        try deviceRequest(createConnection, timeout: timeout)
+    }
+}
+
+// MARK: - Command
+
+/// Create Connection Command
+///
+/// This command causes the Link Manager to create a connection to the remote device with the BD_ADDR specified by the command parameters. This command causes the local BR/EDR Controller to begin the Page process to create a link level connection. The Link Manager will determine how the new ACL connection is established. This ACL connection is determined by the current state of the device, its piconet, and the state of the device to be connected.
+///
+/// - Note:
+/// The Host should enable as many packet types as possible for the Link Manager to perform efficiently. However, the Host must not enable packet types that the local device does not support.
+public struct HCICreateConnection: HCICommandParameter {
+
+    public static let command = LinkControlCommand.createConnection
+    
+    /// BD_ADDR of the Device to be connected.
+    public var address: Address
+    
+    /// The Packet_Type command parameter specifies which packet types the Link Manager shall use for the ACL connection. When sending HCI ACL Data Packets the Link Manager shall only use the packet type(s) specified by the Packet_Type command parameter or the always-allowed DM1 packet type. Multiple packet types may be specified for the Packet Type parameter by performing a bit-wise OR operation of the different packet types. The Link Manager may choose which packet type to be used from the list of acceptable packet types.
+    public var packetType: PacketType
+    
+    /// The Page_Scan_Repetition_Mode parameter specifies the page scan repetition mode supported by the remote device with the BD_ADDR. This is the information that was acquired during the inquiry process.
+    public var pageScanRepetitionMode: PageScanRepetitionMode
+    
+    // Must be set to 0x00
+    public var reserved: Reserved
```

Comment:
This should not be a `public` property, this is a implementation / byte encoding detail.